task/sql-tasks.js

```diff
@@ -178,7 +276,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT 
+            COUNT(UnitsInStock) AS TotalOfCurrentProducts,
+            SUM(Discontinued) AS TotalOfDiscontinuedProducts

```

Comment:
use count when you need to count records, it better shows intent

but it's better to write this query like this
notice that from-clause is optional.



```
select
   (select count()...) as 'T'
   (select count()...) as 'TT'
```